app/models/katello/ext/indexed_model.rb

```diff
@@ -11,10 +11,6 @@ def self.display_attributes
         end
 
         if Rails.env.development? || Rails.env.production?
-          include Tire::Model::Search
```

Comment:
I think this class, IndexedModel, can be removed all together as well as all references to it.